specs/lightclient/beacon-chain.md

```diff
@@ -543,21 +548,23 @@ def process_final_updates(state: BeaconState) -> None:
     # Reset eth1 data votes
     if next_epoch % EPOCHS_PER_ETH1_VOTING_PERIOD == 0:
         state.eth1_data_votes = []
-    # [Added in hf-1] Update sync committees    
+    # [Added in hf-1] Update sync committees
     if next_epoch % EPOCHS_PER_SYNC_COMMITTEE_PERIOD == 0:
         state.current_sync_committee = state.next_sync_committee
         state.next_sync_committee = get_sync_committee(state, next_epoch + EPOCHS_PER_SYNC_COMMITTEE_PERIOD)
     # Update effective balances with hysteresis
     for index, validator in enumerate(state.validators):
-        balance = state.balances[index]
-        HYSTERESIS_INCREMENT = uint64(EFFECTIVE_BALANCE_INCREMENT // HYSTERESIS_QUOTIENT)
-        DOWNWARD_THRESHOLD = HYSTERESIS_INCREMENT * HYSTERESIS_DOWNWARD_MULTIPLIER
-        UPWARD_THRESHOLD = HYSTERESIS_INCREMENT * HYSTERESIS_UPWARD_MULTIPLIER
+        balance_increments = state.balances[index] * HYSTERESIS_QUOTIENT // state.balance_denominator
```

Comment:
((porting previous comments to here))
https://github.com/ethereum/eth2.0-specs/pull/2150/files#r550261538
@dankrad: Suggest name `balance_in_increments` to make it clear that `increment` is menat as a unit here